charts/spire/values.yaml

```diff
@@ -41,10 +41,33 @@ spire-agent:
   enabled: true
   nameOverride: agent
 
+upstream-spire-agent:
```

Comment:
It seems to me that by having the agent and the upstream CA defined in different places, that we are not designing a user interface that concentrates the concerns into the same location.

If I am using nested spire, I should be able to specify the sensible shared items for the upstream agent and upstream CA items under a locally grouped item, the "upstream system".  Minimally I would expect them both to be enabled simultaneously and disabled simultaneously.

Reply:
Updated to be enabled simultaneously.